extensions/wikia/AdEngine/js/provider/gpt/sraHelper.js

```diff
@@ -10,8 +10,15 @@ define('ext.wikia.adEngine.provider.gpt.sraHelper', [
 		sraSlots = [
 			'INVISIBLE_SKIN',
 			'TOP_LEADERBOARD'
+		],
+		optionalSlots = [
+			'INVISIBLE_SKIN'
 		];
 
+	function isOptional(slotName) {
```

Comment:
Please add a description of what it means that a slot is optional.

Reply:
Those are the slots used only in UAP:Roadblock template. I renamed some parts of code to make it clear.